targets/posix/nanoCLR/Memory.cpp

```
@@ -0,0 +1,50 @@
+// See LICENSE file in the project root for full license information.
+//
+
+#include <cstdlib>
+#include <cstring>
+#include <mutex>
+
+namespace
+{
+    constexpr size_t c_DefaultHeapSizeBytes = 10 * 1024 * 1024;
+
+    unsigned char *s_MemoryStart = nullptr;
```

Clang (14.0.6)

[warning] variable 's_MemoryStart' is non-const and globally accessible, consider making it const

(cppcoreguidelines-avoid-non-const-global-variables)